surveys/2021-trademark-survey/README.md

```diff
@@ -0,0 +1,52 @@
+# The Rust 2021 Trademark Survey
+
+The trademark survey is aimed at gathering data about how the community wishes
+the trademark policy to function in practice to help guide the development of
+the Rust tradmark policy.
+
+Discussion of the survey happens on [TODO: get a dedicated channel / stream for trademark survey discussions?](https://rust-lang.zulipchat.com/#narrow/stream/294169-t-community.2Frust-survey-2021).
+
+## Tentative timeline
+
+* **Late September**: High-level design document completed
+* **Mid October**: Survey design completed/translation begins
```

Comment:
How long do you think the translation process will take? If we can get this survey design done early October, could the survey go out late October or early November, potentially?

Reply:
I'm unsure, @rylev do you have any idea how long the translation normally takes?

Reply:
Since it's volunteer driven and we don't have a definitive list of languages, it's hard to say. We could experiment with paid translations, but we've relied on volunteers for other surveys up until this point. 2 weeks would be an absolute minimum I would say.